test/views/pages/map/dynamic_map_test.dart

```diff
@@ -120,9 +125,28 @@ void main() {
       );
       await setPostsFirestore(userPosts, fakeFirestore);
 
+      challenges = [nearbyPosts.first, farPosts.first, userPosts.first];
```

Comment:
I think that the `userPosts.first` post cannot be chosen as challenge in theory as it is one of the current user post. We may want to change it by another 😄